src/components/ManagerPage/AppSearchBar.js

```diff
@@ -62,7 +62,7 @@ class AppSearchBar extends PureComponent<Props, State> {
           }
           renderRight={
             query ? (
-              <Box justify="center" cursor="pointer" onClick={this.reset} px={3}>
+              <Box justify="center" cursor="text" onClick={this.reset} px={3}>
```

Comment:
imo it should be `default` (but it should also have hover state)

Reply:
done